util/libcrypto.num

```diff
@@ -4598,3 +4598,9 @@ EVP_MAC_do_all_sorted                   4551	1_1_2	EXIST::FUNCTION:
 EVP_str2ctrl                            4552	1_1_2	EXIST::FUNCTION:
 EVP_hex2ctrl                            4553	1_1_2	EXIST::FUNCTION:
 EVP_PKEY_supports_digest_nid            4554	1_1_2	EXIST::FUNCTION:
+SRP_VBASE_add0_user                     4555	1_1_2	EXIST::FUNCTION:SRP
```

Comment:
Oups, these SRP* changes comes from a recent master.
Something with your Git client (or its use) is wrong here .